docs/adr/005-ask-user-question-ux-design.md

```
@@ -0,0 +1,128 @@
+
+### 採用: 方式E（行削除による選択）
+
+**Vimの標準的な編集操作をそのまま活用する方式を採用する。**
+
+#### 基本コンセプト
+
+Assistantが質問を提示する際、選択可能な項目をプレーンテキストとして配置する。ユーザーは不要な選択肢を**行削除**（`dd`など）で削除し、最終的に残った選択肢を採用する。
+
+#### 実装アーキテクチャ
+
+**1. Agent Wrapperでのメッセージ変換（サーバー側）**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)